Recruit 타입 참조 버그 해결

## 🐛 어떤 버그인가요?

> 어떤 버그인지 간결하게 설명해주세요

### 버그 재현 방법

69번 PR 병합 후 build 

### 스크린샷 첨부

```zsh
> oz_externship_fe_template@0.0.0 build
> tsc -b && vite build

src/components/recommendSection/RecommendUser.tsx:9:30 - error TS2307: Cannot find module '@/types/interfaceRecruit' or its corresponding type declarations.

9 import type { recruit } from '@/types/interfaceRecruit'
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~

src/testRoutes/testPages/hyejeong/dummyRecommendArray.ts:2:30 - error TS2307: Cannot find module '@/types/interfaceRecruit' or its corresponding type declarations.

2 import type { recruit } from '@/types/interfaceRecruit'
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~

src/types/_pageInterfaces.ts:2:30 - error TS2307: Cannot find module './interfaceRecruit' or its corresponding type declarations.

2 import type { recruit } from './interfaceRecruit'
                               ~~~~~~~~~~~~~~~~~~~~
```

## 🛠️ 수정 방향

> 어떻게 수정해야 하나요?

- [ ] 타입 참조 수정

--- a/src/components/recommendSection/RecommendUser.tsx
+++ b/src/components/recommendSection/RecommendUser.tsx
@@ -3,10 +3,14 @@ import {
   Hstack,
   Vstack,
 } from '@/components/commonInGeneral/layout'
-import type { Lecture, RecommendPageProps, RecommendPageType } from '@/types'
+import type {
+  Lecture,
+  RecommendPageProps,
+  RecommendPageType,
+  Recruit,
+} from '@/types'
 import RecruitCard from '../recruit/RecruitCard'
 import LectureCard from '../lecture/lectureCard/LectureCard'
-import type { recruit } from '@/types/interfaceRecruit'
 import Container from '../commonInGeneral/layout/_Container'
 import Tag from '../commonInProject/tag/Tag'
 import RoundBox from '../commonInGeneral/roundBox/RoundBox'
@@ -43,7 +47,7 @@ const RecommendUser = <T extends RecommendPageType>({
               recommendedArray.map((recommend) => (
                 <RecruitCard
                   key={recommend.uuid}
-                  recruit={recommend as recruit}
+                  recruit={recommend as Recruit}
                 />
               ))}
             {type === 'lecture' &&

--- a/src/testRoutes/testPages/hyejeong/dummyRecommendArray.ts
+++ b/src/testRoutes/testPages/hyejeong/dummyRecommendArray.ts
@@ -1,5 +1,4 @@
-import type { Lecture } from '@/types'
-import type { recruit } from '@/types/interfaceRecruit'
+import type { Lecture, Recruit } from '@/types'
 import img from '@/testRoutes/testPages/nari/d8b6dce9691774374a170cfc1d2c92b1b571c146.jpg'
 
 export const dummyLectureRecommendArray: Lecture[] = [
@@ -71,7 +70,7 @@ export const dummyLectureRecommendArray: Lecture[] = [
   },
 ]
 
-export const dummyRecruitRecommendArray: recruit[] = [
+export const dummyRecruitRecommendArray: Recruit[] = [
   {
     id: 101,
     uuid: 'b0f8e91E-4a33-4e91-bf80-4f4e5e89d123',

--- a/src/types/_pageInterfaces.ts
+++ b/src/types/_pageInterfaces.ts
@@ -1,11 +1,11 @@
 import type { Lecture } from './_lectureInterfaces'
-import type { recruit } from './interfaceRecruit'
+import type { Recruit } from './_recruitInterface'
 
 // 추천섹션
 export type RecommendPageType = 'recruit' | 'lecture'
 
 export type Recommended<T extends RecommendPageType> = T extends 'recruit'
-  ? recruit
+  ? Recruit
   : Lecture
 
 export interface RecommendPageProps<T extends RecommendPageType> {

